Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -3383,7 +3383,34 @@ private void OnPutItemInContainer(uint itemGuid, uint containerGuid, int placeme
             Prev_PutItemInContainer[1] = Prev_PutItemInContainer[0];
             Prev_PutItemInContainer[0] = new PutItemInContainerEvent(itemGuid, containerGuid, placement);
         }
-        
+
+        // Map of the various legendary keys that may be given on emote/quest and the number of uses.
+        // Static map for quick comparisons
+        static Dictionary<uint, int> _LegendaryKeyUses = new Dictionary<uint, int>()
+        {
+            {48746, 1 }, // Aged Legendary Key
```

Comment:
If there is something I can use other than the literal values, please provide an example if that is preferred.

Reply:
WeenieClassName